Yini-He/Weekly_Meetings.md

```diff
@@ -3,7 +3,42 @@
 * [07 July 2019](#date-07-July-2019)
 * [28 June 2019](#date-28-June-2019)
 * [17 June 2019](#date-17-June-2019)
+* [13 June 2019](#date-13-Augest-2019)
```

Comment:
Please put the most recent entry at the top of the list.

Also the links need to be all in lowercase, and they should point to the right headings (not 13 June!)